CHANGELOG.md

```diff
@@ -24,6 +24,11 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
   can be used to more easily introduce automatic formatting changes in a new
   commit separate from other changes.
 
+* `jj workspace add` now accepts a `--sparse-patterns=<MODE>` option, which
+  allows control of the sparse patterns for a newly created workspace: `copy`
+  (inherit from parent; default), `none` (no patterns), or `empty` (the empty
+  working copy pattern).
```

Comment:
There's actually no "empty working copy pattern"; sparse patterns don't support general filesets. That's instead represented by an empty list of paths to include. Relatedly, I still find `none` confusing. I interpret it the same was as `empty`. I think `full` is clearer (what it means in practice is that we use `[RepoPathBuf::root()]`).